helix-core/src/main/java/org/apache/helix/manager/zk/ZkBucketDataAccessor.java

```diff
@@ -233,7 +241,7 @@ public <T extends HelixProperty> boolean compressedBucketWrite(String rootPath,
       }
     };
     if (!_zkBaseDataAccessor.update(rootPath + "/" + LAST_SUCCESSFUL_WRITE_KEY,
```

Comment:
just putting a note here, I think this update() calls also need to be changed to either `create` if first time and then `update` for later updates. Or move to `doUpdate` fn with `ttl` support, based on how the PR discussion concludes.